python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/graphql_helpers.py

```diff
@@ -2,24 +2,40 @@
 from datetime import datetime, timezone
 from typing import Any, Tuple
```

Comment:
I think we agreed that if we changed catalogue we would create a different library rather than reuse this one? Is that right? If so we can drop the parent package of this module and just have `data_platform_catalogue.client`